src/pages/home/report/ReportActionCompose/SilentCommentUpdater/index.tsx

```diff
@@ -16,21 +16,6 @@ function SilentCommentUpdater({comment, commentRef, reportID, value, updateComme
     const {preferredLocale} = useLocalize();
     const prevPreferredLocale = usePrevious(preferredLocale);
 
-    useEffect(() => {
-        /**
-         * Schedules the callback to run when the main thread is idle.
-         */
-        if ('requestIdleCallback' in window) {
-            const callbackID = requestIdleCallback(() => {
-                updateComment(comment ?? '');
-            });
-            return () => cancelIdleCallback(callbackID);
-        }
-        updateComment(comment ?? '');
-
-        // eslint-disable-next-line react-hooks/exhaustive-deps -- We need to run this on mount
-    }, []);
```

Comment:
This is the main fix here, this effect was causing the updateComment to be called with empty string when navigating quickly to a report and we already started to type fast